src/plugins/calibration/include/plugins/calibration/calibration.h

```diff
@@ -135,6 +135,17 @@ class Calibration : public PluginBase {
      */
     void calibrate_magnetometer_async(CalibrateMagnetometerCallback callback);
 
+    /**
+     * @brief Callback type for calibrate_levelhorizon_async.
+     */
+
+    using CalibrateLevelHorizonCallback = std::function<void(Calibration::Result, ProgressData)>;
+
+    /**
+     * @brief Perform level horizon calibration.
+     */
+    void calibrate_levelhorizon_async(CalibrateLevelHorizonCallback callback);
```

Comment:
I'm confused why this is `calibrate_levelhorizon_async`. Given that it is also `LevelHorizon` it should be `calibrate_level_horizon_async`.

I don't see that it is wrong in proto. @JonasVautherin do you understand why that is?

Reply:
You were 15 seconds faster on the comment :laughing: